### PR TITLE
Add null assertion

### LINKED ISSUE #124: Missing null/isNull assertion
Hi. Is there a reason why there is an assertion `notNull` but `isNull`/`null` assertion is missing? I know `same` can be used instead but it is also possible to use `notSame` instead of notNull. Looks like it's not consistent. But I believe there is a reason behind this. Or not?

### PR DESCRIPTION
**Pull request by dosten**
This PR adds support for a `null` assertion, fixes #124

### CODE REVIEW

**Comment by dosten** [comment]
ping @rquadling btw, we use this assertion lib actively at our test suite, can I do something to help with the develop? because the lib seems to be slightly abandoned

File: lib/Assert/Assertion.php [hunk 1]
```diff
@@ -156,15 +158,16 @@ class Assertion
     const INVALID_BOOLEAN           = 13;
     const VALUE_EMPTY               = 14;
     const VALUE_NULL                = 15;
-    const INVALID_STRING            = 16;
```
Comment (rquadling) [inline lib/Assert/Assertion.php]: Can you not renumber the existing constants. Doing so potentially breaks anyone using values rather than constants.

**Comment by rquadling** [comment]
Hi. 1 small change required, to not renumber the constants as this breaks BC for anyone NOT using the constants but the values in any exception handling logic.

File: lib/Assert/Assertion.php (hunk 1 above)
Comment (dosten) [inline lib/Assert/Assertion.php]: What should be a good number for this new constant? I see that 25 is skipped, maybe we can use this number, what do you think?

Comment (rquadling) [inline lib/Assert/Assertion.php]: The numbering thing is odd. I'll address it (if necessary) in a major release, but, for the time being, 25 is available, so seems to be a perfectly reasonable one to use.

**Comment by rquadling** [comment]
Thank you.

**Pull request merged**